Sources/XUI/DeepLink/DeepLinkable.swift

```diff
@@ -7,14 +7,33 @@
 //
 
 public protocol DeepLinkable: AnyObject {
-    @DeepLinkableBuilder var children: [DeepLinkable] { get }
+    var children: [DeepLinkable] { get }
 }
 
 extension DeepLinkable {
 
-    @DeepLinkableBuilder
     public var children: [DeepLinkable] {
-        get {}
+        Mirror(reflecting: self)
+            .values(extractDeepLinkable)
+    }
+
+    private func extractDeepLinkable(from object: Any) -> DeepLinkable? {
+        if let value = object as? DeepLinkable {
+            return value
+        } else if let value = object as? DeepLinkableWrapper {
+            return extractDeepLinkable(from: value.content)
+        } else {
+            return nil
+        }
+    }
+
+}
+
+extension Mirror {
+
+    fileprivate func values<Value>(_ compactMap: (Any) -> Value?) -> [Value] {
+        children.compactMap { compactMap($0.value) }
+            + (superclassMirror?.values(compactMap) ?? [])
```

Comment:
Let's rename the `compactMap` closure as it's not a compact map, it's the function that is applied by compact map (but not only!).

Suggestion: `mappingFunction` or `transform`.

(I know this code was just moved, but I only noticed it now.)

Also, this is a good example where more code is easier to read:

    func values<Value>(_ mappingFunction: (Any) -> Value?) -> [Value] {
        let values = children.compactMap { mappingFunction($0.value) }

        if let superClassValues = superclassMirror?.values(mappingFunction) {
            return values + superClassValues
        } else {
            return values
        }
    }

This makes it much more visible when recursion occurs and which is the recursion anchor.

Reply:
(Also not sure about the function name `values` but I see where it comes from and I don't have a better idea right now.)